runtime/vm/createramclass.cpp

```diff
@@ -1506,7 +1507,11 @@ popFromClassLoadingStack(J9VMThread *vmThread)
  */
 static VMINLINE BOOLEAN
 loadSuperClassAndInterfaces(J9VMThread *vmThread, J9ClassLoader *classLoader, J9ROMClass *romClass, J9Class *elementClass,
-	UDATA packageID, BOOLEAN hotswapping, UDATA classPreloadFlags, J9Class **superclassOut, J9Module *module)
+	UDATA packageID, BOOLEAN hotswapping, UDATA classPreloadFlags, J9Class **superclassOut, J9Module *module
+#if defined(J9VM_OPT_VALHALLA_NESTMATES)
+	, J9Class **nestHostOut
```

Comment:
I'm not overjoyed about splitting the function declaration across three lines like this. Would it be better to rewrite from
```
loadSuperClassAndInterfaces(J9VMThread *vmThread, J9ClassLoader *classLoader, J9ROMClass *romClass, J9Class *elementClass,
UDATA packageID, BOOLEAN hotswapping, UDATA classPreloadFlags, J9Class **superclassOut, J9Module *module)
```
to
```
#if defined(J9VM_OPT_VALHALLA_NESTMATES)
loadSuperClassAndInterfaces(J9VMThread *vmThread, J9ClassLoader *classLoader, J9ROMClass *romClass, J9Class *elementClass,
UDATA packageID, BOOLEAN hotswapping, UDATA classPreloadFlags, J9Class **superclassOut, J9Module *module , J9Class **nestHostOut)
#else /* defined(J9VM_OPT_VALHALLA_NESTMATES) */
loadSuperClassAndInterfacesAndNestHost(J9VMThread *vmThread, J9ClassLoader *classLoader, J9ROMClass *romClass, J9Class *elementClass,
UDATA packageID, BOOLEAN hotswapping, UDATA classPreloadFlags, J9Class **superclassOut, J9Module *module)
#endif /* defined(J9VM_OPT_VALHALLA_NESTMATES) */
```
instead of
```
static VMINLINE BOOLEAN
loadSuperClassAndInterfaces(J9VMThread *vmThread, J9ClassLoader *classLoader, J9ROMClass *romClass, J9Class *elementClass,
	UDATA packageID, BOOLEAN hotswapping, UDATA classPreloadFlags, J9Class **superclassOut, J9Module *module
#if defined(J9VM_OPT_VALHALLA_NESTMATES)
	, J9Class **nestHostOut
#endif /* defined(J9VM_OPT_VALHALLA_NESTMATES) */
	)
```
or is there an alternate approach?

Reply:
Actually, looking at the code, there are some places where we have duplicated function headers doing the same here would be consistent.

Reply:
For what it's worth, my default eclipse auto-indent handles the first case correctly but mishandles the second case by adding an extra indent for all lines following. I'm leaning towards the first option again because of that convenience.

Reply:
Alternate option: Call a `loadNestHost` function in the same section of code as `loadSuperClassAndInterfaces`.
- Should still occur early enough to avoid the shared vscratch table problem.
- Does not involve changing function header
- Keeps the function name accurate (ie. `loadSuperClassAndInterfaces` does not indicate that the nest is being loaded)
- Does create some code duplication